server/views/pages/visit/confirmUpdate.njk

```diff
@@ -0,0 +1,56 @@
+{% extends "layout.njk" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageHeaderTitle = "Confirm update" %}
+{% set pageTitle %}
+{% if errors | length %}Error: {% endif %}{{ applicationName }} - {{ pageHeaderTitle }}
+{% endset %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        {% include "partials/errorSummary.njk" %}
+        <h1 class="govuk-heading-l">
+          This visit is in less than {{ policyNoticeDaysMin }} {{ "day" | pluralise(policyNoticeDaysMin) }}.
+          Do you want to update the booking?
+        </h1>
+        <p>
+          Changes made at this stage may require additional actions.
+          For example, if operational reports have been printed, they may need to be printed again.
+        </p>
+    </div>
+</div>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <form action="/visit/{{ reference }}/update/confirm-update" method="POST" novalidate>
+            <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+            {% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+            {{ govukRadios({
+              name: "confirmUpdate",
+              items: [
+                {
+                  value: "true",
```

Comment:
Slightly tempted to prefer making the values `"yes" | "no"` to avoid any potential confusion - i.e. it's a `string` value and not a `boolean`. Personal preference though: this works and is fine 🤷‍♂️